appveyor.yml

```diff
@@ -12,15 +12,38 @@ environment:
   CYGMIRROR: http://ftp.inf.tu-dresden.de/software/windows/cygwin32
   CYGCACHE: C:\cygwin64\var\cache\setup
   opam_url: https://github.com/fdopen/opam-repository-mingw/releases/download/0.0.0.1/opam64.tar.xz
+  DESTCOQ: C:\coq64_inst
+  matrix:
+  - USEBUILDSCRIPT: false
+  - USEBUILDSCRIPT: true
 
 install:
-- cmd: '%CYGROOT%\setup-x86_64.exe -qnNdO -R %CYGROOT% -l %CYGCACHE% -s
-  %CYGMIRROR% -P rsync -P patch -P diffutils -P make -P unzip -P m4 -P findutils -P time'
-- cmd: '%CYGROOT%/bin/bash -l %APPVEYOR_BUILD_FOLDER%/dev/build/windows/appveyor.sh'
+- cmd: 'if %USEBUILDSCRIPT% == false (%CYGROOT%\setup-x86_64.exe -qnNdO -R %CYGROOT% -l %CYGCACHE% -s
+  %CYGMIRROR% -P rsync -P patch -P diffutils -P make -P unzip -P m4 -P findutils -P time)'
+- cmd: 'if %USEBUILDSCRIPT% == false (%CYGROOT%/bin/bash -l -c "/usr/bin/timeout -s SIGTERM -k 1s 30.0s curl --insecure --retry 2 --head \"https://github.com\"")'
+- cmd: 'if %USEBUILDSCRIPT% == false (%CYGROOT%/bin/bash -l %APPVEYOR_BUILD_FOLDER%/dev/build/windows/appveyor.sh)'
```

Comment:
In passing, let me remark that I don't really get how it makes sense to have the `appveryor.sh` script live in `dev/build/windows/` and not instead in `dev/ci`.